openapi/paths/balance-transactions@{id}.yaml

```
@@ -5,15 +5,15 @@ get:
     - Users
   tags: ["Balance transactions"]
   summary: Retrieve a balance transaction
-  description: |
-    Retrieve a balance transaction with specified identifier.
+  description: |-
+    Retrieves a balance transaction with a specified identifier.
 
-    This operation is experimental and not guaranteed to be backward compatible.
+    > **Important:** This operation is experimental and may change.
   operationId: GetBalanceTransaction
   x-sdk-operation-name: get
   responses:
     '200':
-      description: An entry was retrieved.
+      description: Balance transaction entry retrieved.
```

```suggestion
      description: Balance transaction retrieved.
```

Not sure if the word `entry` adds anything here?